### ISSUE TITLE
Markdown code blocks generated outside DoxyCode block in Latex output

### ISSUE THREAD

**Issue by TransientResponse** [copperspice/doxypress#16]
Take the example markdown file here:
```markdown
# Usage with Python

The DCS-100 SDK can be used in Python code with either [IronPython](http://www.ironpython.net) or [Python.NET](http://pythonnet.github.io). The usage is a little different between the two.

## IronPython

~~~{.py}
import os
import clr
...
```

The generated Latex for this markdown will be something like:

```latex
The D\+C\+S-\/100 S\+DK can be used in Python code with either \href{http://www.ironpython.net}{\tt Iron\+Python} or \href{http://pythonnet.github.io}{\tt Python.\+N\+ET}. The usage is a little different between the two.

\subsection*{Iron\+Python}


\begin{DoxyCode}
\end{DoxyCode}

1 \textcolor{keyword}{import} os
2 \textcolor{keyword}{import} clr
3 ...
```

Additionally, it would be nice if the latex output used Minted to format such code, as it would avoid the issues of pre-processing the code (for which I will file another issue I found) for DoxyPress and turn it into a straight-through transfer.

**Comment by agserm** [comment]
Thanks to your clear and repeatable report, we have been able to track this down. This will be fixed in the upcoming release of DoxyPress.

**Comment by TransientResponse** [comment]
Great, thanks! Awesome turnaround time on these issues. I really appreciate your work on these.

**Comment by TransientResponse** [comment]
Confirmed fixed in 1.2.10.
